### PR TITLE
fix: broken manifest for `nfs-server.yaml`

### PR DESCRIPTION
**Pull request by mayankshah1607**
Signed-off-by: Mayank Shah <mayankshah1614@gmail.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: examples/kubernetes/nfs-provisioner/nfs-server-lb.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ spec:
                     - mountPath: /nfsshare
                       name: data-volume
             volumes:
-                name: data-volume
-                hostPath: /nfsshare-volume
-                type: DirectoryOrCreate
\ No newline at end of file
+                - name: data-volume
+                  hostPath:
+                    path: /nfsshare-volume
```
Comment (andyzhangx) [inline examples/kubernetes/nfs-provisioner/nfs-server-lb.yaml:42]: expose `/exports` ?

**Comment by andyzhangx** [comment]
could you also add https://github.com/kubernetes-csi/csi-driver-smb/blob/master/hack/verify-yamllint.sh check into this repo? thanks.

File: examples/kubernetes/nfs-provisioner/nfs-server-lb.yaml [hunk 2]
```diff
@@ -37,6 +37,7 @@ spec:
                     - mountPath: /nfsshare
                       name: data-volume
             volumes:
-                name: data-volume
-                hostPath: /nfsshare-volume
-                type: DirectoryOrCreate
\ No newline at end of file
+                - name: data-volume
+                  hostPath:
+                    path: /exports
```
Comment (mayankshah1607) [inline examples/kubernetes/nfs-provisioner/nfs-server-lb.yaml:42]: @andyzhangx I have made the change, thanks

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Comment by andyzhangx** [comment]
pls add set release-note `none` next time, thanks.

**Pull request merged**